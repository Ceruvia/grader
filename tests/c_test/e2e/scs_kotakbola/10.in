703 797 621
H 23
M 24
M 21
B 90
B 50
H 33
H 89
H 4
H 2
M 70
M 7
M 71
M 58
M 65
B 83
H 24
H 89
M 100
B 38
B 14
B 45
M 28
H 93
M 71
H 67
B 67
H 25
M 12
H 59
B 94
H 88
H 63
H 67
E
