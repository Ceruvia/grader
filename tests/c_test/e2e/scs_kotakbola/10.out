726 527 481
Tidak
